Added ability to set consumer inactive threshold parameter

flyDuckWithMac opened the pull request.

nekufa commented:
@flyDuckWithMac thanks for the merge request! 
please, fix code style issues (`phpcs` binary can be used).

flyDuckWithMac commented:
I run cs fixer with default project config:

tests folder:
```
 ./vendor/bin/php-cs-fixer fix tests
PHP CS Fixer 3.68.5 Persian Successor by Fabien Potencier, Dariusz Ruminski and contributors.
PHP runtime: 8.3.6
Running analysis on 1 core sequentially.
You can enable parallel runner and speed up the analysis! Please see usage docs for more information.
Loaded config default from "/home/user/PhpstormProjects/extProjects/nats.php/.php-cs-fixer.php".
Using cache file ".php-cs-fixer.cache".
Paths from configuration file have been overridden by paths provided as command arguments.
 23/23 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%


Fixed 0 of 23 files in 0.186 seconds, 18.00 MB memory used
```

src folder:

```
 ./vendor/bin/php-cs-fixer fix src
PHP CS Fixer 3.68.5 Persian Successor by Fabien Potencier, Dariusz Ruminski and contributors.
PHP runtime: 8.3.6
Running analysis on 1 core sequentially.
You can enable parallel runner and speed up the analysis! Please see usage docs for more information.
Loaded config default from "/home/user/PhpstormProjects/extProjects/nats.php/.php-cs-fixer.php".
Using cache file ".php-cs-fixer.cache".
Paths from configuration file have been overridden by paths provided as command arguments.
 48/48 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%


Fixed 0 of 48 files in 0.310 seconds, 18.00 MB memory used
```

nekufa commented:
@flyDuckWithMac, sorry, there are no rules for extra spaces for constant definition (there should be no allignment).
in addition, consumer limits constants should be public, so they can be used to define consumer limits.
this is simillar to other nats client configuration:
```php
->setConsumerLimits([ConsumerLimits::INACTIVE_THRESHOLD => $oneSecondInNanoseconds])
```

flyDuckWithMac commented:
> @flyDuckWithMac, sorry, there are no rules for extra spaces for constant definition (there should be no allignment). in addition, consumer limits constants should be public, so they can be used to define consumer limits. this is simillar to other nats client configuration:
> 
> ```
> ->setConsumerLimits([ConsumerLimits::INACTIVE_THRESHOLD => $oneSecondInNanoseconds])
> ```

Ok, its fixed

nekufa commented:
@flyDuckWithMac thanks!
